ocs_ci/ocs/constants.py

```diff
@@ -200,6 +200,10 @@
 OSD_PVC_GENERIC_LABEL = "ceph.rook.io/DeviceSet"
 CEPH_ROOK_IO_PVC_LABEL = 'ceph.rook.io/pvc'
 
+# Auth Yaml
+BUCKETNAME = 'ocs-ci-data'
```

Comment:
I think it might be better to give this constant a more specific name, like `OCSCI_DATA_BUCKET`

Reply:
all review comment fixed.